arch/arm64/src/imx9/imx9_trdc_config.h

```diff
@@ -0,0 +1,285 @@
+/****************************************************************************
+ * arch/arm64/src/imx9/imx9_trdc_config.h
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.  The
+ * ASF licenses this file to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the
+ * License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ ****************************************************************************/
+
+#define SP(X)           ((X) << 12)
+#define SU(X)           ((X) << 8)
+#define NP(X)           ((X) << 4)
+#define NU(X)           ((X) << 0)
+#define LK              BIT(31)
+#define RWX             7
+#define RW              6
+#define RX              5
+#define R               4
+#define X               1
+#define TEE_SHM_SIZE    0x0
+
+/* GLBAC7 is used for TRDC only, any setting to GLBAC7 will be ignored
+ * GLBAC6 is used for fused modules, any setting to GLBAC6 will be ignored
+ */
+
+/* aonmix */
+
+struct trdc_glbac_config trdc_a_mbc_glbac[] =
```

Comment:
Are the constants in the struct board specific? I.e. should be under board..?

Reply:
Yes they are kind of board specific, and changing structs to const means almost re-writing .c file, but it can be done of course. Maybe we can include imx9_trdc_config.h to board.h

Reply:
#include "imx9_trdc.h"
#include <arch/board/imx9_trdc_config.h>   This seems to work, but then this patch need to be splitted so that _config.h goes to board git tree